pos_payment_terminal/static/src/js/payment_terminal.js

```diff
@@ -20,56 +20,60 @@ odoo.define('pos_payment_terminal.payment', function (require) {
             return this._oca_payment_terminal_pay();
         },
 
-
         _oca_payment_terminal_pay: function () {
             var self = this;
             var order = this.pos.get_order();
             var pay_line = order.selected_paymentline;
             var currency = this.pos.currency;
             if (pay_line.amount < 0) {
```

Comment:
maybe ``<=`` here, don't you think ?

Reply:
Done in https://github.com/akretion/pos/pull/20